Localization: big docs update (#8961)
DevExpress/devextreme-documentation: concepts/Common/Localization and Globalization/10 Value Formatting/10 Format UI Component Values/20 Custom Format String.md

    <tr>
    <td>&#35;</td>
    <td>
            Any number of leading digits, a single digit, or nothing. If this character goes first in the format string, it can match multiple leading digits (before the decimal point). Subsequent characters match a single digit. If the formatted number does not have a digit in the corresponding position, it displays nothing.<br/>

>             For example, if you apply format "#0.#" to "123.45", the result is "123.5".

Reviewer: format what?

>             For example, if you apply format "#0.#" to "123.45", the result is "123.5".

Author: Maybe:
```suggestion
            For example, if you format "123.45" with "#0.#", the result is "123.5".
```

    </tr>
    <tr>
    <td>;</td>
        <td>    Separates positive and negative format patterns.  <br/>
          For example, the "#0.##;(#0.##)" format displays a positive number according to the pattern before the semicolon (";"), and a negative number according to the pattern after the semicolon (";").<br/>

>             If you do not use this character and the additional pattern, negative numbers display a minus ("-") prefix.

Reviewer: ```suggestion
            If you do not use this character , nor the additional pattern, negative numbers display a minus ("-") prefix.
```

    // Add a group separator
    format: ",##0.###" // 123,456.789
    

> The examples below show different ways to apply percentage formatting to decimals. Use caution if your format string starts with `0` because the formatted number may lose leading digits.

Reviewer: ```suggestion
The examples below demonstrate different options used to apply percentage formatting to decimals. Use caution if your format string starts with `0` because the formatted number may lose leading digits.
```
